pages/events/_id/edit.vue

```diff
@@ -238,8 +238,15 @@ export default {
           },
           {
             name: 'price',
-            placeholder: 'Examples: 10EUR, 5USD, FREE',
-            description: 'Examples: 10EUR, 5USD, FREE',
+            type: 'price',
+            placeholder: 'Examples: 10, 5, 0',
```

Comment:
let's remove examples, there is no need cuz now there is no way to write it wrong

Reply:
👍